pallets/transaction-multi-payment/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = '2018'
 homepage = 'https://github.com/galacticcouncil/hydra-dx'
 license = 'Unlicense'
 repository = 'https://github.com/galacticcouncil/hydra-dx'
-version = '2.0.0'
```

Comment:
Semver?

Reply:
not sure if we should even use this approach for migrations... we already use a special storage key in the claims pallet

Reply:
Ideally migration will be a part of PR that introduces the breaking change so it would be bumped there. The question is what to do if it slips? (shouldn't happen) But if yes, we're fixing a breaking change so it should also increase the main version? We can ignore this now but after migrating to the released version we should definitely make it a rule.

Reply:
we have to change the version if we want to use an embedded storage key for migrations

Reply:
Now the question is... Do we also increase version of runtime? I'd say, yes...? (We already bumped spec version once to "quickfix" but it should have been bumped as a part of this really, we can either bump again to keep runtime version synced with spec_version as it is the same thing, or we can ignore bumping runtime version for this one case to do the same thing.

Reply:
yeah, we should probably bump everything that is dependent on this pallet

Reply:
Ok let's not bump runtime this time, but we should think about automating this for the next versions.